mlflow/tracking/_tracking_service/client.py

```diff
@@ -571,6 +571,14 @@ def _record_logged_model(self, run_id, mlflow_model):
             )
         self.store.record_logged_model(run_id, mlflow_model)
 
+    def _get_artifact_repo_for_trace(self, trace_id):
+        trace_info = self.get_trace_info(trace_id)
+        artifact_uri_attr = next(a for a in trace_info.attributes if a.key == "artifact_uri")
```

Comment:
@harupy if you don't mind, can we update the backend so that attribute is called `mlflow.artifactLocation` and update the client too?